src/Subclasses/User.php

```diff
@@ -22,6 +22,14 @@ class User extends ParseUser implements Authenticatable
 {
     use CastsParseProperties;
 
+	/**
+     * @return string
+     */
+    public function getAuthIdentifierName()
+    {
+        return $this->getUsername();
```

Comment:
Ahhhhhh, I get it.  You're right.  I see the change you made.  I just read unique identifier and thought 'username'.  I just pulled the update and see you did it correctly.  Really appreciate it.  

This was my very first open source fork/push.  Hope I get better as I continue forward :)